projects/shared/src/lib/_mocked/mocked-renderer-2.ts

```diff
@@ -0,0 +1,8 @@
+export class MockRenderer2 {
```

Comment:
I'm following a convention with this name, which is to prefix the name of the thing being mocked with the word "Mock" (and the filename prefix 'mocked-').  Is it a good idea to break this convention?  The person reading the code would have to check the `providers` block in the `spec.ts` to confirm the actual name of the thing that was being mocked, if it were to be changed to that it could no longer be deduced.

Reply:
I get it after the explanation you gave me about this object make sense to be named `MockRenderer2`